pegasus/sites.v3/code.org/views/csf_congrats.haml

```diff
@@ -13,49 +13,7 @@
     :text=> I18n.t(:"just_did_course_donor", donor_twitter: get_random_donor_twitter, course: course_name)
   }
 
-  rec_code_studio = {
-    "course1" => "course2",
-    "course2" => "course3",
-    "course3" => "course4",
-    "accelerated" => "course4",
-    "course4" => "applab",
-
-    ScriptConstants::COURSEA_NAME => ScriptConstants::COURSEB_NAME,
-    ScriptConstants::COURSEB_NAME => ScriptConstants::COURSEC_NAME,
-    ScriptConstants::PRE_READER_EXPRESS_NAME => ScriptConstants::COURSEC_NAME,
-    ScriptConstants::COURSEC_NAME => ScriptConstants::COURSED_NAME,
-    ScriptConstants::COURSED_NAME => ScriptConstants::COURSEE_NAME,
-    ScriptConstants::COURSEE_NAME => ScriptConstants::COURSEF_NAME,
-    ScriptConstants::COURSEF_NAME => "applab",
-    ScriptConstants::EXPRESS_NAME => "applab",
-
-    ScriptConstants::COURSEA_2018_NAME => ScriptConstants::COURSEB_2018_NAME,
-    ScriptConstants::COURSEB_2018_NAME => ScriptConstants::COURSEC_2018_NAME,
-    ScriptConstants::PRE_READER_EXPRESS_2018_NAME => ScriptConstants::COURSEC_2018_NAME,
-    ScriptConstants::COURSEC_2018_NAME => ScriptConstants::COURSED_2018_NAME,
-    ScriptConstants::COURSED_2018_NAME => ScriptConstants::COURSEE_2018_NAME,
-    ScriptConstants::COURSEE_2018_NAME => ScriptConstants::COURSEF_2018_NAME,
-    ScriptConstants::COURSEF_2018_NAME => "applab",
-    ScriptConstants::EXPRESS_2018_NAME => "applab",
-
-    ScriptConstants::COURSEA_2019_NAME => ScriptConstants::COURSEB_2019_NAME,
-    ScriptConstants::COURSEB_2019_NAME => ScriptConstants::COURSEC_2019_NAME,
-    ScriptConstants::PRE_READER_EXPRESS_2019_NAME => ScriptConstants::COURSEC_2019_NAME,
-    ScriptConstants::COURSEC_2019_NAME => ScriptConstants::COURSED_2019_NAME,
-    ScriptConstants::COURSED_2019_NAME => ScriptConstants::COURSEE_2019_NAME,
-    ScriptConstants::COURSEE_2019_NAME => ScriptConstants::COURSEF_2019_NAME,
-    ScriptConstants::COURSEF_2019_NAME => "applab",
-    ScriptConstants::EXPRESS_2019_NAME => "applab",
-
-    ScriptConstants::COURSEA_2020_NAME => ScriptConstants::COURSEB_2020_NAME,
-    ScriptConstants::COURSEB_2020_NAME => ScriptConstants::COURSEC_2020_NAME,
-    ScriptConstants::COURSEC_2020_NAME => ScriptConstants::COURSED_2020_NAME,
-    ScriptConstants::COURSED_2020_NAME => ScriptConstants::COURSEE_2020_NAME,
-    ScriptConstants::COURSEE_2020_NAME => ScriptConstants::COURSEF_2020_NAME,
-    ScriptConstants::COURSEF_2020_NAME => "applab",
```

Comment:
Did we lose the course f and express recommendations above?

Reply:
I don't think so, aren't those cases included in `test_csf_next_course_recommendation`?

Reply:
Did you mean csf_next_course_recommendation or test_csf_next_course_recommendation? I don't see them in csf_next_course_recommendation

Reply:
Oh nevermind I missed they were above the list. Got it